backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/migrations/V0003_AddFirstActivationDate.kt

```diff
@@ -0,0 +1,26 @@
+package app.ehrenamtskarte.backend.migration.migrations
+
+import app.ehrenamtskarte.backend.migration.Migration
+import app.ehrenamtskarte.backend.migration.Statement
+
+/**
+ * Add firstActivationDate column to cards.
+ * Set current_timestamp for firstActivationDate for existing active cards since we have no firstActivationDate for them.
+ */
+@Suppress("ClassName")
+internal class V0003_AddFirstActivationDate() : Migration() {
+    override val migrate: Statement = {
+        exec(
+            """
+            ALTER TABLE cards ADD "firstActivationDate" TIMESTAMP NULL;
+            """.trimIndent()
+        )
+        exec(
+            """
+            UPDATE cards
+            SET "firstActivationDate" = current_timestamp
+            where "totpSecret" is not null
```

Comment:
I am not entirely sure, whether semicolon is necessary here.

```suggestion
            where "totpSecret" is not null;
```